### PR TITLE
Reorganization of BIDS website

### PR DESCRIPTION
**Pull request by guiomar**
Hi! 

I have done an attempt of reorganizing website content. 
Some edits are still needed. I'll also work on some figures

I tried to condense main information in 6 tabs:
1) home (general info and pointers to specific tabs)
2) BIDS (BIDS general info, what it is, acknoledgements, etc)
3) COLLAB (get started, get involved, code of conduct)
4) SPECS (link to specs, and BEPs table)
5) TOOLS (validator, general (pyBIDS+BIDS-Matlab), BIDS Apps, converters)
6) TEAM (governance, etc)

I have a few technical questions @robertoostenveld @melanieganz would love to meet, just let me know when best fits you?

### CODE REVIEW

**Comment by melanieganz** [comment]
Ha, you are much faster than me, @guiomar! We could try to meet this week some time during the evenings when we are anyways online for OHBM? Or should we postpone it to next week? @robertoostenveld how doe your calendar look like?

**Comment by robertoostenveld** [comment]
we could indeed meet online. Any evening from 19:30 or 20:00 onward, except tonight. We can arrange this tonight, or on slack.

File: assets/img/BIDS_Organizational_Design.png
Comment (Remi-Gau) [inline assets/img/BIDS_Organizational_Design.png:1]: I like the color version of this figure: we should probably try to grab it for the new website

**Comment by Remi-Gau** [comment]
may probably worth checking if there is something to salvage from this PR for the new website

**Comment by Remi-Gau** [comment]
Will close the PR but we can always find some of the content if needed by checking the PR.

**Pull request closed without merging**